internal/consensus/pbts_test.go

```diff
@@ -159,29 +167,36 @@ func (p *pbtsTestHarness) observedValidatorProposerHeight(ctx context.Context, t
 
 	p.currentHeight++
 	incrementHeight(p.otherValidators...)
-	return res
+	return res, rs.ProposalBlock.Time
 }
 
 func (p *pbtsTestHarness) height2(ctx context.Context, t *testing.T) heightResult {
 	t.Helper()
 	signer := p.otherValidators[0].PrivValidator
-	height3BlockTime := p.height2ProposedBlockTime.Add(-blockTimeIota)
-	return p.nextHeight(ctx, t, signer, p.height2ProposalDeliverTime, p.height2ProposedBlockTime, height3BlockTime)
+	return p.nextHeight(ctx, t, signer,
+		p.firstBlockTime.Add(p.height2ProposalTimeDeliveryOffset),
+		p.firstBlockTime.Add(p.height2ProposedBlockOffset),
+		p.firstBlockTime.Add(p.height2ProposedBlockOffset+10*blockTimeIota))
 }
 
 func (p *pbtsTestHarness) intermediateHeights(ctx context.Context, t *testing.T) {
 	t.Helper()
 	signer := p.otherValidators[1].PrivValidator
-	blockTimeHeight3 := p.height4ProposedBlockTime.Add(-blockTimeIota)
-	p.nextHeight(ctx, t, signer, blockTimeHeight3, blockTimeHeight3, p.height4ProposedBlockTime)
+	p.nextHeight(ctx, t, signer,
+		p.firstBlockTime.Add(p.height2ProposedBlockOffset+10*blockTimeIota),
+		p.firstBlockTime.Add(p.height2ProposedBlockOffset+10*blockTimeIota),
+		p.firstBlockTime.Add(p.height4ProposedBlockOffset))
```

Comment:
This change is odd to me, although not tweaked when cherry-picking.

We are replacing subtracting `blockTimeIota` by summing `10*blockTimeIota`.